### PR TITLE
refactor(docs): add components roadmap

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,70 @@ See the [open issues](https://github.com/vonage/vivid-3/issues) for a full list
 
 <br>
 
+
+### Components roadmap
+| name                 | to do      | status      |
+| -------------------- | ---------- | ----------- |
+| button               | migrate    | done        |
```
Comment (rachelbt) [inline README.md:46]: I personally wouldn't call the button done because the "stacked button" is still not in main

File: docs/components.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Components
+
+<!-- Statuses can be GA | Alpha | WIP | backlog | deprecated -->
```
Comment (yinonov) [inline docs/components.md:3]:
```suggestion
<!-- Statuses can be GA | alpha | WIP | backlog | deprecated | obsolete -->
```

File: docs/components.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# Components
+
+<!-- Statuses can be GA | alpha | WIP | backlog | deprecated | obsolete -->
+
+| Name   | vivid | vivid@next |
```
Comment (rinaok) [inline docs/components.md:5]: why the button is duplicated?

File: docs/components.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+| button               |       GA        |     GA     |
+| badge                |       GA        |     GA     |
+| icon                 |       GA        |     GA     |
+| layout               |      alpha      |     GA     |
+| elevation            |      alpha      |     GA     |
+| side drawer          |      alpha      |     WIP    |
+| popup                |      alpha      |     WIP    |
+| anchor               |      alpha      |     GA     |
+| breadcrumbs          |       WIP       |     WIP    |
+| surface              |    obsolete     | deprecated |
```
Comment (yinonov) [inline docs/components.md:16]: other way around no?

**Review by yinonov — CHANGES_REQUESTED** [review]

File: docs/components.md (hunk 4 above)
Comment (yinonov) [reply docs/components.md:16]:
```suggestion
```

**Review by yinonov — CHANGES_REQUESTED** [review]

File: docs/components.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+# Components
+
+<!-- Statuses can be GA | alpha | WIP | backlog | deprecated | obsolete -->
+
+| Name                 | VIVID           | VIVID@next |
+| -------------------- | --------------- | ---------- |
+| button               |       GA        |     GA     |
```
Comment (yinonov) [inline docs/components.md:7]:
```suggestion
| button               |       GA        |     alpha     |
```

**Review by yinonov — CHANGES_REQUESTED** [review]

File: docs/components.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+| badge                |       GA        |     GA     |
+| icon                 |       GA        |     GA     |
+| layout               |      alpha      |     GA     |
+| elevation            |      alpha      |     GA     |
+| side drawer          |      alpha      |     WIP    |
+| popup                |      alpha      |     WIP    |
+| anchor               |      alpha      |     GA     |
```
Comment (yinonov) [inline docs/components.md:14]:
```suggestion
| anchor               |      -      |     WIP     |
```

**Review by yinonov — CHANGES_REQUESTED** [review]

File: docs/components.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+| fab                  |      alpha      | -          |
+| snackbar             |       GA        | -          |
+| notched outline      |       GA        | -          |
+| switch               |       GA        | -          |
+| tab                  |       GA        | -          |
+| tab bar              |       GA        | -          |
+| textarea             |       GA        | -          |
+| textfield            |       GA        | -          |
+| drawer               |      alpha      | -          |
+| chip                 |      alpha      | -          |
```
Comment (yinonov) [inline docs/components.md:41]:
```suggestion
| chip                 |      obsolete      | -          |
```

**Review by yinonov — CHANGES_REQUESTED** [review]

File: docs/components.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+| fab                  | alpha    | -          |
+| snackbar             | GA       | -          |
+| notched outline      | GA       | -          |
+| switch               | GA       | -          |
+| tab                  | GA       | -          |
+| tab bar              | GA       | -          |
+| textarea             | GA       | -          |
+| textfield            | GA       | -          |
+| drawer               | alpha    | -          |
+| chip                 | obsolete |deprecated  |
```
Comment (yinonov) [inline docs/components.md:40]:
```suggestion
| chip                 | obsolete | -          |
```

File: docs/components.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+| chipset              | obsolete |deprecated  |
```
Comment (yinonov) [inline docs/components.md:41]:
```suggestion
| chipset              | obsolete | -          |
```

File: docs/components.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+| textfield            | GA       | -          |
+| drawer               | alpha    | -          |
+| chip                 | obsolete | -          |
+| chipset              | obsolete |deprecated  |
+| top app bar fixed    | GA       | -          |
+| top app bar          | GA       | -          |
+| file picker          | GA       | -          |
+| banner               | GA       | -          |
+| audio player         | GA       | -          |
+| carousel             | obsolete | deprecated |
```
Comment (yinonov) [inline docs/components.md:47]:
```suggestion
| carousel             | obsolete | -          |
```

File: docs/components.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+| button-toggle-group  | GA       | -          |
+| empty state          | GA       | -          |
+| datagrid             | GA       | -          |
+| helper message       | GA       | -          |
+| note                 | GA       | -          |
+| calendar             | GA       | -          |
+| accordion            | alpha    | -          |
+| card                 | alpha    | -          |
+| list expansion panel | GA       | -          |
+| datepicker           | alpha    | deprecated |
```
Comment (yinonov) [inline docs/components.md:58]:
```suggestion
| datepicker           | alpha    | -          |
```

**Review by yinonov — APPROVED** [review]

**Pull request merged**